js/src/features/project/selectors.ts

```diff
@@ -1,7 +1,6 @@
 import { RootState } from "MyTypes";
```

Comment:
I propose always to use `select` instead `get`. I have reed some sources and it is de facto standard.

Reply:
Done